### DISCUSSION TITLE
How to turn off makrodnw syntax warnings?

### DISCUSSION THREAD

**Discussion by Abrillant-Lee** [LazyVim/LazyVim#2556] [Q&A] [answered]
I don’t want to have a bunch of grammatical and spelling errors that it falsely reports when I write a markdown file.How to close this!
![image](https://github.com/LazyVim/LazyVim/assets/129536239/1b9c8706-cf42-443e-b8dc-519e2bcc9441)

**Comment by dpetka2001** [comment, marked as answer]
Try `<leader>ud` maybe?

**Reply by Abrillant-Lee** [reply]
Thanks for you advice! I tried `<leader>ud` command and it did exactly what i wanted.🥰

![image](https://github.com/LazyVim/LazyVim/assets/129536239/d045902a-d6f1-4299-8ba0-eb08eb5cda80)
